packages/js/github-actions/actions/phpcs-diff/README.md

```diff
@@ -35,3 +35,13 @@ jobs:
       - name: Run PHPCS to changed lines of code
         uses: woocommerce/grow/phpcs-diff@actions-v1
 ```
+
+#### Specify the PHP version:
+
+```yaml
+steps:
+  - name: Run PHPCS to changed lines of code
+    uses: woocommerce/grow/phpcs-diff@actions-v1
+    with:
+      php-version: 16
```

Comment:
:crystal_ball:  That's being prepared for a far-future ;)

Reply:
😂 Thanks for catching this. Fixed with another typo in https://github.com/woocommerce/grow/pull/25/commits/51a3c9fe7930f409be554ac6dd3d2750dc11d742.